free-programming-books-it.md

```diff
@@ -47,6 +47,9 @@
 ### BASH
 
 * [Guida avanzata per la bash](http://www.dmi.unict.it/diraimondo/web/wp-content/uploads/classes/so/mirror-stuff/abs-guide.pdf) (PDF)
+* [Guida per principianti bash](http://codex.altervista.org/guidabash/guidabash_1_11.pdf) (PDF)
+* [PC Assembly Language](http://drpaulcarter.com/pcasm/) - Paul A. Carter
```

Comment:
I don't know why git sign as newline (maybe because i have sort link in alphabetical order) but the line 51 (PC Assembly..) It is not my addition